### ISSUE TITLE
Incomplete coloring with larger --mindim values.

### ISSUE THREAD

**Issue by kepppis** [satoshiiizuka/siggraphasia2019_remastering#31]
I've run couple times into the situation where the coloring is perfect if mindim is less than 450, but with larger values the colorization will fade and the output is with pale colors which are very far from reference images.  Tested this with two environments, one with Nvidia Tesla k40 with 11 GB ram and pytorch 1.2.0 (the last version compatible with oldish Tesla card).  
Other with Nvidia Quadro P5000 with 16 GB ram and pytorch 1.8.1. The source video and the reference pics are 576p and the only thing I am changing is the mindim value.

**Comment by kepppis** [comment]
Here are sample videos of the problem - same source and same reference pictures:

https://youtu.be/Qfq6L0l4dk8 --mindim 540

https://youtu.be/OjyTnmFb_PY --mindim 320

The red color is the always the first to disappear and fade.

**Comment by vitacon** [comment]
@kepppis Have you managed to solve this? It's a shame it seems the authors lost insterest in this project.

**Comment by kepppis** [comment]
Unfortunately not, but I've found a way to get pretty similar results to highres coloring.  I do the coloring on lower resolution and overlay it over the highres bw image using suitable color blending filters.

**Comment by vitacon** [comment]
@kepppis Yeah, that's a classic trick. =) Unfortunately the "color mask" is still too blurry to my taste, color details are lost {eyes, lips, etc.} and colors are bleeding. 

![017819](https://user-images.githubusercontent.com/58292841/221955072-1800155f-8d8a-416f-b719-b481a02352dc.png)

I wondered if it would be possible to force a higher working resolution and maybe retrain the model but I guess I'll start from a different place.

**Comment by kepppis** [comment]
Let me know if you find anything usable.

**Comment by vitacon** [comment]
Well, I tried EbSynth for that and I use my own tool Tinterr (eriador.condak.cz/tools.php), but to get a decent result you have to feed it with a LOT of keyframes...

**Comment by vitacon** [comment]
My test result (about 1560 frames and 23 key frames) can be seen here:
https://www.youtube.com/watch?v=D9bcxx4xL00

None of this methods is perfect...

**Comment by kepppis** [comment]
Is it possible to have instructions how to use PNG-Tinterr ?

**Comment by vitacon** [comment]
> Is it possible to have instructions how to use PNG-Tinterr ?

Yes, but I have to create a provisional English manual first... I should have created it a long time ago. =]

**Comment by vitacon** [comment]
@kepppis Well, here is something that might help you:
http://eriador.condak.cz/tools/png-tinterr-manual-eng.htm
